templates/city.html

```diff
@@ -56,39 +56,36 @@ <h1 class="display-4">Ford GoBike Metrics</h1>
           <h4>Most Popular Stations</h4>
           <ul>
             {{ range .MostPopularStations -}}
-            <li>{{ .Station.Name }} ({{ .Count }})</li>
+            <li><a href="http://www.openstreetmap.org/?mlat={{ .Station.Latitude }}&mlon={{ .Station.Longitude }}&zoom=14">{{ .Station.Name }}</a> ({{ .Count }})</li>
```

Comment:
I wanted to link to the official Ford GoBike [map](https://member.fordgobike.com/map/), but it doesn't give you the option of linking to a specific station.